### ISSUE TITLE
import dialog can be used only once - it does not open on a follow up import

### ISSUE THREAD

**Issue by DeepDiver1975** [owncloud/calendar#214]

**Comment by georgehrke** [comment]
this one is really strange, I can't reproduce it all the time

**Comment by georgehrke** [comment]
Ah, it won't work when you select the exact same set of files, because the on change event won't be triggered.
Very low priority for now
